jdi-light-vuetify/src/main/java/com/epam/jdi/light/vuetify/interfaces/MayContainButtons.java

```diff
@@ -0,0 +1,49 @@
+package com.epam.jdi.light.vuetify.interfaces;
+
+import com.epam.jdi.light.common.JDIAction;
+import com.epam.jdi.light.elements.common.UIElement;
+import com.epam.jdi.light.elements.complex.WebList;
+import com.epam.jdi.light.elements.interfaces.base.ICoreElement;
+import com.epam.jdi.light.vuetify.elements.common.VuetifyButton;
+
+public interface MayContainButtons extends ICoreElement {
+	/**
+	 * Gets element's buttons
+	 *
+	 * @return list of buttons
+	 */
+	@JDIAction("Get '{name}' button")
+	default WebList buttons() {
```

Comment:
Why do you get web list and then always cast to VuetifyButton? Why do you not use List< VuetifyButton> instead of WebList?